### PR TITLE
Webpage animation

### PR DESCRIPTION
**Pull request by Paskab012**
### Popup menu displaying
- In this section of the project, the popup menu will be showing once the user clicks on the `See my projects` button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -300,6 +175,193 @@ <h3 class="about-myself-connect">LET'S CONNECT</h3>
...
+            </div>
+            <div class="first-image">
+              <img src="./asset/Snapshoot(1).png" alt="" />
+            </div>
+            <p>
+              Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it to make a type specimen book. It has
+              survived not only five centuries, but also the leap into
```
Comment (Cathella) [inline index.html]: Data in the pops ups is meant to come from the `datas` array in the `app.js` it should not be added there manually 🤔. 

- [ ] Kindly make changes to dynamically place the objects in the corresponding projects dynamically.

File: index.html [hunk 2]
```diff
@@ -300,6 +175,193 @@ <h3 class="about-myself-connect">LET'S CONNECT</h3>
...
+            </div>
+            <div class="tonic-content-canopy">
+              <p>CANOPY</p>
+              <ul class="tonic-list">
+                <li>Back End Dev</li>
+                <li>2021</li>
+              </ul>
+            </div>
+            <div class="first-image">
+              <img src="./asset/Snapshoot(2).png" alt="" />
```
Comment (Cathella) [inline index.html]: - [ ] I suggest that you try to design the popup to follow the design for both desktop and mobile version from template in Figma. You can check out the comparison below and try to make changes accordingly

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @team 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Github Flow.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @KABIKA681, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**